software/controller/lib/core/nvparams.cpp

```diff
@@ -36,8 +36,10 @@ static Debug::Variable::UInt32 dbg_reinit(
     "nvparams_reinit", Debug::Variable::Access::ReadWrite, 0, "",
     "Set to 1 to request a reinit of NV params on next boot.");
 
-static Debug::Variable::UInt32 dbg_serial("serial_number", Debug::Variable::Access::ReadWrite, 0,
-                                          "", "Serial number of the ventilator, in EEPROM");
+/// \TODO: use String type to allow alphanumeric?
+static Debug::Variable::UInt32 dbg_serial("0_controller_serial_number",
```

Comment:
This is actually the device's serial number, which has little to do with controller, as such, this might be misleading, maybe something like `0_ventilator_serial_number`?

Reply:
Fair enough. Changed.